v1/lib/consoleOutput.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * @file
+ * @version 0.1
+ * @copyright 2016 CN-Consult GmbH
+ * @author Max Späth <max.spaeth@cn-consult.eu>
+ */
+
+require_once 'baseoutput.php';
+
+/**
+ * This class implements the output function from BaseOutput and prints the gamefield every cycle
+ * to the console.
+ */
+class ConsoleOutput extends BaseOutput
+{
+    /**
+     * This function is extended from the baseoutput class.
+     * It prints the gamefield to the console for the number of cycles given.
+     *
+     * @param GameFieldController $_gameFieldController Object of gamefieldcontroller.
```

Comment:
Think about it again? What you need to write here? Right! You need to write what for this parameter is needed...!